app/controllers/api/v0/admin/configs_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V0
+    module Admin
+      class ConfigsController < ApiController
+        before_action :authenticate!
```

Comment:
I'm not sure we should enable OAuth2 here for now, maybe we can use `authenticate_with_api_key_or_current_user!` ?

Reply:
Good call.